styles/style.css

```diff
@@ -22,13 +22,22 @@ button {
   font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
   color: var(--grey-dark);
   font-size: 1rem;
+  
 }
 
-h1,h2,h3,h4,h5,h6 {
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   margin-top: 0;
   margin-bottom: 0;
 }
 
+h2 {
+  margin-top: 60px;
+}
```

Comment:
How do you think we could apply this to the learn more h2 _without_ adding it to the uncoming events h2? So that we wont have the gap at the top sidebar